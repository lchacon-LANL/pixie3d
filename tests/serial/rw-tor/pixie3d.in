 &datin
!    E0=  0.000000000000000E+000 ,  0.000000000000000E+000 ,  2.499914999999999E-006
    ilevel   = 1
    test     = t

    nxd      = 32
    nyd      = 32
    nzd      = 1

    xmax     = 1d0
    ymax     = 0d0
    zmax     = 0d0

    numerical_grid = f

    bcs      = 'spt','rsw','per','per','per','per'

    tau_w    = 1d2
    
    eta      = 0d0  !1.d-9
    nu       = 1d0 !1.d-9
    dd       = 0d0
    chi      = 0d0

    equil    = 'gstor'
    gstor_I_drive = t
    coords   = 'tor'
    gparams  = 1.65d0,1d0,1d0,2d0,0d0,1.5151d0,0.01d0,0.d0
!    gparams  = 10d0,1d0,1d0,2d0,0d0,1.5151d0,0.01d0,0.d0

    nh2      = 1
    nh3      = 1

    gamma    = 1.6666666667d0

    dt       = 1d-1
    ndstep   = 1
    numtime  = 1

!    pvx      = 1d-10
    pvx      = 0d0
    odd      = f
    random   = f

    cnfactor = 0.
    sm_flag  = 2

    advect   = 4
    v_advect = 4

    restart  = f
    timecorr = t

    source   = f

    pc_type     = 's1'
    pc_tol      = 1d-1
    pc_max_iter = 1
    pc_debug    = f
    pc_debug_it = 0
    pc_divclean = f
    lag_pinch_bc = f

    maxksp   = 50
    maxitgm  = 50
    maxitnwt = 7
    tolgm    = 8d-1
    rtol     = 1d-3
    atol     = 0d0
    method   = 1
    mf_eps   = 1d-6
    vol_wgt  = t

    nc_eom_f = t
    nc_eom_v = t
    solve_rho= f
!    adiabatic= t

    check_grid = f
 /

&graphics
    sel_diag  = 2,5,6,7,9,10,14,22,23,24
    dplot = 1d-4
!    prof_conf = 1,'r',1,16,1
    prof_conf = 1,'r',1,1,1
    cont_conf = 3,'R','Z',1,1,1
    hdf_plot = t
    clean    = f
    poincare    = t
    adios = t
 /

