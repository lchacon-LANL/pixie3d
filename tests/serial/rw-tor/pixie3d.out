
  BIM: setting BCs
  BIM: allocating vars
  BIM: setting BCs
  BIM: computing matrix
 BIM DIAG theta Bng Rg Zg |J.ng|
 BIM DIAG   9.8174770424681035E-002   1.7416767638575528        2.6451847266721966        9.8017140329560604E-002   2.6454273641918498     
 BIM DIAG  0.29452431127404310        1.7188172919843434        2.6069403357322090       0.29028467725446233        2.6071736653088768     
 BIM DIAG  0.49087385212340517        1.6728838042020515        2.5319212643483553       0.47139673682599764        2.5321363309747746     
 BIM DIAG  0.68722339297276724        1.6033985822958767        2.4230104533627372       0.63439328416364549        2.4231989956215929     
 BIM DIAG  0.88357293382212931        1.5095180753424811        2.2843932841636456       0.77301045336273688        2.2845480504081110     
 BIM DIAG   1.0799224746714913        1.3898666429702342        2.1213967368259978       0.88192126434835494        2.1215117649910158     
 BIM DIAG   1.2762720155208533        1.2422977237762618        1.9402846772544624       0.95694033573220882        1.9403555270989421     
 BIM DIAG   1.4726215563702154        1.0635891273616065        1.7480171403295608       0.99518472667219682        1.7480410681962040     
 BIM DIAG   1.6689710972195775       0.84913695002477441        1.5519828596704395       0.99518472667219693        1.5519589278407606     
 BIM DIAG   1.8653206380689396       0.59287597671215408        1.3597153227455376       0.95694033573220894        1.3596444381416681     
 BIM DIAG   2.0616701789183018       0.28804788498214468        1.1786032631740022       0.88192126434835505        1.1784881433453513     
 BIM DIAG   2.2580197197676641       -6.9786852018847323E-002   1.0156067158363542       0.77301045336273688        1.0154517835792776     
 BIM DIAG   2.4543692606170264      -0.47513751716000213       0.87698954663726270       0.63439328416364515       0.87680075789103895     
 BIM DIAG   2.6507188014663887      -0.89872804771483095       0.76807873565164453       0.47139673682599703       0.76786334818887569     
 BIM DIAG   2.8470683423157510       -1.2732130195539466       0.69305966426779075       0.29028467725446150       0.69282595695052496     
 BIM DIAG   3.0434178831651133       -1.5002530199140449       0.65481527332780287        9.8017140329559507E-002  0.65457222727119480     
 BIM DIAG   3.2397674240144756       -1.5002530199140431       0.65481527332780320       -9.8017140329561908E-002  0.65457222727119513     
 BIM DIAG   3.4361169648638379       -1.2732130195539419       0.69305966426779153      -0.29028467725446383       0.69282595695052573     
 BIM DIAG   3.6324665057132002      -0.89872804771482395       0.76807873565164575      -0.47139673682599920       0.76786334818887692     
 BIM DIAG   3.8288160465625625      -0.47513751715999702       0.87698954663726414      -0.63439328416364704       0.87680075789104062     
 BIM DIAG   4.0251655874119248       -6.9786852018842230E-002   1.0156067158363560      -0.77301045336273833        1.0154517835792793     
 BIM DIAG   4.2215151282612871       0.28804788498214834        1.1786032631740044      -0.88192126434835616        1.1784881433453536     
 BIM DIAG   4.4178646691106493       0.59287597671215819        1.3597153227455401      -0.95694033573220960        1.3596444381416704     
 BIM DIAG   4.6142142099600116       0.84913695002477818        1.5519828596704421      -0.99518472667219715        1.5519589278407628     
 BIM DIAG   4.8105637508093739        1.0635891273616096        1.7480171403295635      -0.99518472667219660        1.7480410681962071     
 BIM DIAG   5.0069132916587362        1.2422977237762649        1.9402846772544655      -0.95694033573220794        1.9403555270989452     
 BIM DIAG   5.2032628325080985        1.3898666429702364        2.1213967368260005      -0.88192126434835338        2.1215117649910185     
 BIM DIAG   5.3996123733574608        1.5095180753424822        2.2843932841636483      -0.77301045336273466        2.2845480504081133     
 BIM DIAG   5.5959619142068231        1.6033985822958783        2.4230104533627390      -0.63439328416364249        2.4231989956215951     
 BIM DIAG   5.7923114550561854        1.6728838042020524        2.5319212643483571      -0.47139673682599403        2.5321363309747764     
 BIM DIAG   5.9886609959055477        1.7188172919843452        2.6069403357322098      -0.29028467725445822        2.6071736653088782     
 BIM DIAG   6.1850105367549100        1.7416767638575525        2.6451847266721975       -9.8017140329556093E-002   2.6454273641918502     
 BIM: solving system
 BIM: computing error
 BIM error:          32   4.0252970693706617E-003
